homeassistant/components/evohome/strings.json

```
@@ -50,7 +50,7 @@
       "fields": {
         "entity_id": {
           "name": "Entity",
-          "description": "The entity_id of the zone."
+          "description": "The entity ID of the zone."
```

```suggestion
          "description": "[%key:component::evohome::services::set_zone_override::fields::entity_id::description%]"
```
Let's also do the name